client/app/visualizations/chart/chart-editor.html

```diff
@@ -99,7 +111,7 @@
       </ui-select>
     </div>
 
-    <div class="checkbox" ng-if="options.globalSeriesType != 'custom'">
+    <div class="checkbox" ng-if="options.globalSeriesType != 'custom' && options.globalSeriesType != 'heatmap'">
```

Comment:
Replace it with `"['custom', 'heatmap'].indexOf(options.globalSeriesType) >= 0"` - it's easier to read and modify in future